src/bin/transparent_main_window.rs

```diff
@@ -15,7 +15,7 @@ use std::env::args;
 
 fn build_ui(application: &gtk::Application) {
     let window = ApplicationWindow::new(application);
-    set_visual(&window, &None);
+    set_visual(&window, None);
```

Comment:
This is btw one of the reasons why it's nicer to have the reference inside the `Option`